### PR TITLE
Gnosis/gnosis safe - Post Fix & Contract Clean up

### PR DESCRIPTION
**Pull request by NickGonzalez04**
### What changes were made?

- Completion of function hot fixes.
- Contract Code clean up.
- Function naming clean up for brevity.
- Structured contract layout to follow Solidity style guide principles.
- Added NatSpec Format (function details) for function purpose and input parameters.

### Background info on changes:

- Audit prep for transition hand off of code commit and documentation (post team review).
- Completion of official `v1.0.0`.
- Completion to carry out contract function testing.

Details of Contract Updates can be found at:
Team Shared Drive => Engineering => `Smart Contract Version Change-log`.

### CODE REVIEW

File: scripts/4_Signature_Test.js [hunk 1]
```diff
@@ -57,30 +57,30 @@ async function main() {
...
+    //     WorkDetails: workdetails,
+    //     // WerkExpert: workExpert
+    // },
+    //     primaryType: "WorkDetails",
+    //     domain: domainData,
+    //     message: workObj
+    // });
+
+    // let hashedMsg = ethers.utils.hashMessage(data)
+    // console.log(hashedMsg)
```
Comment (ivyastrix) [inline scripts/4_Signature_Test.js:76]: remove if no longer needed

File: scripts/1_local_deploy.js [hunk 2]
```diff
@@ -14,38 +16,45 @@ async function main() {
   const deployedTwaliContract = await TwaliImp.deploy();
   await deployedTwaliContract.deployed();
 
-  console.log("Twali Implementation contract deployed to:", deployedTwaliContract.address);
+  // console.log("Twali Implementation contract deployed to:", deployedTwaliContract.address);
```
Comment (ivyastrix) [inline scripts/1_local_deploy.js:19]: remove if no longer needed

File: scripts/1_local_deploy.js [hunk 3]
```diff
@@ -14,38 +16,45 @@ async function main() {
...
-    1663196020
-  );
+  // console.log("Twali Clone contract deployed to:", deployedFactory.address);
+  // // Create test clone contracts with test data
+  // const newlocalContract = await deployedFactory.createTwaliClone(
+  //   "https://contract-metadata.s3.amazonaws.com/v1/<bucketObjectReferenceUrl>.json",
+  //   "3000000000000000000",
+  //   "150000000000000000",
+  //   1663196020,
+  //   1663196020
```
Comment (ivyastrix) [inline scripts/1_local_deploy.js:35]: remove if no longer needed

File: scripts/1_local_deploy.js [hunk 4]
```diff
@@ -14,38 +16,45 @@ async function main() {
...
  
- let instance = await hre.ethers.getContractAt('TwaliContract', newlocalContract);
- console.log(instance);
-  // await deployedFactory.createTwaliClone("<enter URI string or just normal string data>");
+//  let instance = await hre.ethers.getContractFactory('TwaliContractFactory', owner);
+// let myInstance = await hre.ethers.utils.;
 
+// console.log(myInstance);
+
+  // await deployedFactory.createTwaliClone("<enter URI string or just normal string data>");
```
Comment (ivyastrix) [inline scripts/1_local_deploy.js:44]: remove if no longer needed

File: scripts/1_local_deploy.js [hunk 5]
```diff
@@ -14,38 +16,45 @@ async function main() {
...
 //   console.log("Contract Clones created:", getClones);
 
-//   // Example getting a clone Contract by its address
+// //   // Example getting a clone Contract by its address
 //   const clone1 = await hre.ethers.getContractAt("TwaliContract", getClones[0]);
 
-//   const metaData = await clone1.contract_sowMetaData();
-//   console.log(metaData);
+  // const metaData = await clone1.getWorkDetails(owner, 0);
+  // console.log(metaData);
```
Comment (ivyastrix) [inline scripts/1_local_deploy.js:57]: remove if no longer needed

**Review by ivyastrix — COMMENTED** [review]
mostly just some copy editing, otherwise looks good!

**Review by NickGonzalez04 — COMMENTED** [review]
All changes suggested removed and addressed. Doing a commit and push to reflect changes.

File: contracts/Twali/TwaliContract.sol [hunk 6]
```diff
@@ -281,7 +285,7 @@ contract TwaliContract is Initializable, ReentrancyGuard {
     }
 
 
-
+// TODO: [] - Deparacate function
```
[the remark this answers is not in the document]
Comment (NickGonzalez04) [reply contracts/Twali/TwaliContract.sol]: Same as the Deposit event. Was removed most likely not part of PR. Sorry!

**Pull request merged**